measurement-kit-test/NetworkMeasurement.mm

```diff
@@ -113,13 +112,20 @@ -(void) run {
                 } catch (...) {
                     /* FALLTHROUGH */
                 }
+                // Here you should update the progress log
```

Comment:
Because I'm printing the log received without modifying it, I think it's not necessary to maintain the previous lines.